pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -255,9 +242,29 @@ func (c *Reconciler) makeBuild(t *v1alpha1.Task, tr *v1alpha1.TaskRun) (*buildv1
 		},
 		Spec: *t.Spec.BuildSpec,
 	}
-	return c.BuildClientSet.BuildV1alpha1().Builds(tr.Namespace).Create(b)
-}
 
-func (c *Reconciler) now() metav1.Time {
-	return metav1.Now()
+	build, err := resources.AddInputResource(b, t, tr, c.resourceLister, logger)
+	if err != nil {
+		logger.Errorf("Failed to create a build for taskrun: %v due to input resource error", err)
```

Comment:
is the `err` meant to be appended here vs. displayed in the middle of the string?